daml-script/daml3/Daml/Script/Questions/Submit.daml

```diff
@@ -50,10 +51,22 @@ submitMustFailInternal actAs readAs disc cmds =
 
 -- This will error if the submission fails.
 
+#ifdef DAML_SCENARIOS
+
 instance HasSubmit Script Commands where
   submit p cmds = submitInternal [p] [] [] cmds
   submitMustFail p cmds = submitMustFailInternal [p] [] [] cmds
 
+#else
```

Comment:
with the other changes, this branch becomes unused, but it's easier to leave it in place in case we need to support daml3-script in daml 2.x